cura/PrinterOutput/Models/MaterialOutputModel.py

```diff
@@ -28,6 +28,7 @@ def getMaterialFromDefinition(guid, type, brand, name):
                             "abs-wss1"	:{"name" :"absr_175"      ,"guid": "88c8919c-6a09-471a-b7b6-e801263d862d"},
                             "asa" 		:{"name" :"asa_175"       ,"guid": "416eead4-0d8e-4f0b-8bfc-a91a519befa5"},
                             "nylon-cf" 	:{"name" :"cffpa_175"     ,"guid": "85bbae0e-938d-46fb-989f-c9b3689dc4f0"},
+                             "nylon12-cf": {"name": "nylon12-cf", "guid": "3c6f2877-71cc-4760-84e6-4b89ab243e3b"},
```

Comment:
`nylon12-cf_175`
following the same pattern

Reply:
```suggestion
                             "nylon12-cf": {"name": "nylon12-cf_175", "guid": "3c6f2877-71cc-4760-84e6-4b89ab243e3b"},
```